### PR TITLE
Slows down SCPs slightly.

### PR DESCRIPTION
**Pull request by JarlElbow**

### CODE REVIEW

**Comment by kachnov** [comment]
I think they should be slightly faster, otherwise they won't ever catch anybody.

**Comment by rhicora** [comment]
For this weekend, we're doing an equal-speed test and seeing if it feels okay. It's very possible @kachnov is right, in which case we can make humans tire out after extended running.

**Review by rhicora — APPROVED** [review]

**Comment by UndeadNappist** [comment]
Kachnov is half-right since different players act and move at different speeds inconsistently, even though there's no good reason.

On the other hand, for some SCPs - those with minions, instant transportation and/or an effective deathtouch - them being slower then players might be a good idea if you assume they have some good way of cornering them.

I strongly suspect for speed testing if the SCPs are NPC controlled they should be faster, but on PC control should be slower, even though that's arbitrary.

File: code/modules/SCP/SCPs/SCP-049.dm [hunk 1]
```diff
@@ -152,7 +152,7 @@ GLOBAL_LIST_EMPTY(scp049_1s)
 
 	if (!(target in orange(1, src)))
 		// moves slightly faster than humans
-		walk_to(src, target, 1, 0.9+config.run_speed)
+		walk_to(src, target, 1, config.run_speed)
```
Comment (JakeDaBoss) [inline code/modules/SCP/SCPs/SCP-049.dm]: Alright, after serious consideration, I'm not going to merge this on SCP's being the same speed as humans. They might not have been in CB, but this is SS13. Balance is everything. I agree 0.9, almost an entire point, faster is too much, but we should see what it does on 0.2 first. If you could make SCP-106 that as well, please.

0.2 looks like a good starting point to nerf or buff further based on results, but to my understanding, 0.2 will give people a fighting chance to run and use airlocks to their advantage.

**Review by JakeDaBoss — CHANGES_REQUESTED** [review]

**Comment by JarlElbow** [comment]
Fixed

**Review by JakeDaBoss — APPROVED** [review]

**Pull request merged**